jetty-server/src/main/java/org/eclipse/jetty/server/ResponseWriter.java

```diff
@@ -456,11 +456,29 @@ public PrintWriter format(Locale l, String format, Object... args)
     { 
         try 
         {
+            
+            if(l==null) 
+                l = _locale;
```

Comment:
probably comment that if the passed locale is null then use any locale set on the response as the default.
Also rename `l` to `locale` for readability